README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
```

Comment:
The command `task dev` to run the server and asset build in watch mode may benefit from an explanation of what exactly it does and any prerequisites that need to be met prior to running it (e.g. configuration files needed, environment variables).